src/pilot/deployment-validation/sanity_test.sh

```diff
@@ -642,6 +643,80 @@ setup_cinder(){
   done
 }
 
+setup_manila(){
+  info "### Manila test"""
+  
+  set_admin_scope
+  manila_exists=$(manila service-list | grep manila-share |  head -n 1  | awk '{print $4}')
+  echo "$manila_exists";
+  if [ "$manila_exists" != 'manila-share' ]; then
+    info "### Manila is not deployed. Skipping..."
+    return 1
+  fi
+
+  info "### Create manila share network (unity requires)..."
+  execute_command "manila share-network-list"  
+  manila_share_network_exists=$(manila share-network-list | grep unity_share_net | awk '{print $4}')
+  if [ "$manila_share_network_exists" != "unity_share_net" ]; then
+    net_id=$(openstack network list | grep " $TENANT_NETWORK_NAME " | head -n 1 | awk '{print $2}')
+    subnet_id=$(openstack network list | grep $TENANT_NETWORK_NAME | head -n 1 | awk '{print $6}')
+    
+    execute_command "manila share-network-create --neutron-net-id  $net_id --neutron-subnet-id $subnet_id --name unity_share_net" 
+  fi   
+  execute_command "manila share-network-list"
+  
+	  
+  execute_command "manila list"
+
+  info "### Kicking off share creation..."
+
+  shares=()
+  share_name=$BASE_SHARE_NAME
+  echo "Share Name = $share_name";
```

Comment:
no need for this line

Reply:
Will remove. I guess I had these for debugging